sema/harvest/__main__.py

```
@@ -159,15 +159,18 @@
 
 
 def make_service(args: argparse.Namespace) -> Harvest:
-    store_info: list = args.store or []
+    store_info: list | None = args.store or None
     log.debug(f"{store_info=}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)